lib/core/collections.toit

```diff
@@ -1022,9 +1022,9 @@ interface ByteArray:
   /**
   Creates a new byte array of the given $size.
 
-  All elements are initialized to 0.
+  All elements are initialized to the $filler, which default to 0.
   */
-  constructor size/int:
+  constructor size/int filler/int=0:
```

Comment:
I would prefer if the filler is a named argument.
At the moment named arguments are passed after positional ones (but in sorted order). Not sure if we want to rely on this for primitive functions, or if it's better to forward to a static function instead.

Also not sure, if `--fill_value` or `--initial_value` wouldn't be better.